tests/unit/amo/components/TestVPNPromoBanner.js

```diff
@@ -44,15 +59,48 @@ describe(__filename, () => {
     );
   };
 
-  it('renders nothing when the feature is disabled', () => {
-    const _config = getFakeConfig({ enableFeatureVPNPromo: false });
+  const setupForComponent = ({
+    clientApp = CLIENT_APP_FIREFOX,
+    regionCode = 'US',
+  } = {}) => {
+    dispatchClientMetadata({ store, clientApp, regionCode });
+  };
```

Comment:
We usually put this in `render()`. Why is a new function needed in this file?

Reply:
My thinking was that this is called to explicitly set up the component to show the banner. I wanted it to be obvious in tests that we are setting the test up to show the banner, and then overriding specific things. We could accomplish the same thing by moving it into render and passing the override values into render instead of calling this function, as you suggest. I was concerned that that might be a bit less explicit, but we are already defaulting `variant` inside render, so I guess that would be no worse. I'll change it.